Src/ILGPU/VariableView.cs

```diff
@@ -40,7 +40,9 @@ namespace ILGPU
         /// <param name="baseView">The base view.</param>
         public VariableView(ArrayView<T> baseView)
         {
-            Trace.Assert(baseView.IsValid && baseView.Length == 1, "Invalid base view");
+            Trace.Assert(
+                baseView.IsValid && baseView.Length == 1,
```

Comment:
MINOR: `Length` is now of type `long`.

```suggestion
                baseView.IsValid && baseView.Length == 1L,
```